operator/pkg/translate/translate_test.go

```diff
@@ -34,8 +34,9 @@ func TestProtoToValuesV13(t *testing.T) {
 		{
 			desc: "default success",
 			yamlStr: `
-meshConfig:
-  rootNamespace: istio-system
+values:
+  global:
+    istioNamespace: istio-system
```

Comment:
I thought this was coming from the IOP `metadata.namespace`?

Reply:
it is - this is just added to the values yaml that's passed to helm. the reason the ns gets added there after parsing IOP is because internally during rendering we don't pass the entire IOP object around but just the spec, so there's no other place to preserve the top level namespace when this is passed in to helm rendering.